<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcPhotoManagerWidget</class>
 <widget class="QWidget" name="MusicLrcPhotoManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌手写真管理</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>605</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌手写真管理</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="artNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手：</string>
   </property>
  </widget>
  <widget class="QLabel" name="artTextLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>43</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>605</width>
     <height>395</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QScrollArea" name="viewArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>585</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>355</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>355</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>355</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>artNameLabel</zorder>
  <zorder>artTextLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
